services/core/Ambient/tests/test_ambient_agent.py

```diff
@@ -124,6 +124,7 @@
                                                        "test_ambient_data.json"
                                                        " variable app_key")
 
+pytestmark = [pytestmark_app, pytestmark_app, pytestmark_locs]
```

Comment:
shouldn't this be pytestmark_api, pytestmark_app, pytestmark_locs

Reply:
Yes, that was a typo. Will change.